packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -54,6 +54,22 @@ async function promiseRaceWithWinner<T>(promises: Promise<T>[]): Promise<{ index
     });
 }
 
+// Gate that when set to "1", instructs to fetch the binary format snapshot from the spo.
+function gatesBinaryFormatSnapshot() {
+    try {
+        if (typeof localStorage === "object" && localStorage !== null) {
+            if  (localStorage.binaryFormatSnapshot === "1") {
+                return true;
+            }
+            if  (localStorage.binaryFormatSnapshot === "0") {
```

Comment:
I'd remove this IF and always return false here. Otherwise, if you want to keep current logic, you can remove check for "1" - true is default value (which does not seem right)